index.html

```diff
@@ -14,6 +14,7 @@
     <script src="src/components/app.js"></script>
     <script src="src/components/header.js"></script>
     <script src="src/components/movies.js"></script>
+    <script src="src/components/actions.js"></script>
```

Comment:
Don't know why this isn't showing up in the `step-6 PR`. Will need to be in there as well.